scripts/update_synapse_database

```diff
@@ -56,6 +75,12 @@ if __name__ == "__main__":
         required=True,
         help="A database config file for either a SQLite3 database or a PostgreSQL one.",
     )
+    parser.add_argument(
+        "--run-background-updates",
+        type=bool,
+        required=False,
+        help="Whether to run background updates after upgrading the database schema.",
```

Comment:
```suggestion
        help="run background updates after upgrading the database schema",
```

Reply:
Update help text for background updates flag.